README.md

```diff
@@ -417,7 +417,7 @@ For many examples on how to use WDL see [the WDL site](https://github.com/broadi
 
 # Configuring Cromwell
 
-Cromwell's default configuration file is located at `src/main/resources/application.conf`.
+Cromwell's default configuration file is located at `core/src/main/resources/reference.conf`.
```

Comment:
Let's chat on slack or something as I don't understand what you're getting at. When I read what you wrote as-is I'd say "yeah, exactly" to it :)